docs/cloud/alerts-notifications/add-mattermost-notification-configuration.md

```diff
@@ -0,0 +1,51 @@
+# Add Mattermost notification configuration
+
+From the Cloud interface, you can manage your space's notification settings and from these you can add a specific configuration to get notifications delivered on Mattermost.
+
+## Prerequisites
+
+To add Mattermost notification configurations you need
```

Comment:
Hey Joao! A `:` at the end maybe?

Reply:
Sure, it makes sense. I will add on all of them.